### PR TITLE
Begins the cleanup of PDA code 

### PR DESCRIPTION
**Pull request by jcatjmeow**
### Intent of your Pull Request

As title.

PDA code is so ass you guys there is legit a segment that is like "if 1 return 1" smh anyway I cleaned up mainly Redd's edgy 666's and the consequences of that, now it's number fucking 5, cause it's the fifth one, god
#### Changelog

:cl:  
spellcheck: The PDA is slightly less shitcode now. Only slightly. 
/:cl:

### CODE REVIEW

**Comment by JamieD1** [comment]
Why not do the whole lot in one go?

**Comment by jcatjmeow** [comment]
Atomize atomize atomize atomize

File: code/game/objects/items/devices/PDA/PDA.dm [hunk 1]
```diff
@@ -56,7 +56,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 	#define FONT_VT "font-family:\"VT323\", monospace;letter-spacing:1px;"
 	#define MODE_MONO 0
 	#define MODE_SHARE 1
-	#define MODE_ORBITRON 2
+	#define MODe_ORBITRON 2
```
Comment (Bibby0110) [inline code/game/objects/items/devices/PDA/PDA.dm]: what the FUCK is this?

**Comment by jcatjmeow** [comment]
Sorry about that folks. No idea how those weird typos got there. But they're all gone now!

**Comment by JamieD1** [comment]
> Sorry about that folks. No idea how those weird typos got there. But they're all gone now!

This been tested?

**Comment by jcatjmeow** [comment]
mhm, It's been tested! But then the uppercase E became a lowercase e like wtf is up with that

**Review by JamieD1 — APPROVED** [review]

**Comment by alexkar598** [comment]
To e or to E

**Pull request merged**